### PR TITLE
New version: REALiX.HWiNFO version 7.60

### LINKED ISSUE #113393: [Update Request]: REALiX.HWiNFO 7.60
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

REALiX.HWiNFO

### Package Version

7.60

### Please describe the changes you would like to see

New Release: 7.60
Download Page: https://www.hwinfo.com/download/

### PR DESCRIPTION
**Pull request by CoolPlayLin**
### Pull request has been created with [Komac](https://github.com/russellbanks/Komac) v1.9.1 :rocket:
- Resolve #113393
 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/113441&drop=dogfoodAlpha

### CODE REVIEW

**Comment by CoolPlayLin** [comment]
It has been trying for a week!

**Comment by AsciiWolf** [comment]
You can use an official mirror:

https://www.sac.sk/download/utildiag/hwi_760.exe

**Comment by AsciiWolf** [comment]
@CoolPlayLin

You can use an official mirror:

https://www.sac.sk/download/utildiag/hwi_760.exe

File: manifests/r/REALiX/HWiNFO/7.60/REALiX.HWiNFO.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2023-07-27
+ElevationRequirement: elevatesSelf
+Installers:
+- Architecture: x86
+  InstallerUrl: https://www.sac.sk/download/utildiag/hwi_760.exe
+  InstallerSha256: D67BCA7B63DDFC5670B88417124BAF37BAE6D8E902A5757693EDEA2FC0F28B00
```
Comment (stephengillie) [inline manifests/r/REALiX/HWiNFO/7.60/REALiX.HWiNFO.installer.yaml:20]:
```suggestion
  InstallerSha256: 97C33C893D9F98F78C7A3BA322A952EE886A210C84D05C2012F9E8CE8C98AF5E
```

File: manifests/r/REALiX/HWiNFO/7.60/REALiX.HWiNFO.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- Architecture: x64
+  InstallerUrl: https://www.sac.sk/download/utildiag/hwi_760.exe
+  InstallerSha256: D67BCA7B63DDFC5670B88417124BAF37BAE6D8E902A5757693EDEA2FC0F28B00
```
Comment (stephengillie) [inline manifests/r/REALiX/HWiNFO/7.60/REALiX.HWiNFO.installer.yaml:23]:
```suggestion
  InstallerSha256: 97C33C893D9F98F78C7A3BA322A952EE886A210C84D05C2012F9E8CE8C98AF5E
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**